app/assets/javascripts/discourse/lib/ajax-error.js.es6

```diff
@@ -32,7 +32,7 @@ export function extractError(error, defaultMessage) {
     } else if (parsedJSON.error) {
       parsedError = parsedJSON.error;
     } else if (parsedJSON.failed) {
-      parsedError = parsedJSON.message;
+      parsedError = parsedJSON.failed;
```

Comment:
careful ... this is changing internal behaviour, I guess we kind of mucked up not having this documented in a test. 

https://github.com/discourse/discourse/blob/7b253dbe4a92f7656bbc779d68e07ec93c56dfed/app/controllers/admin/backups_controller.rb#L40-L40

Reply:
I have fixed it now.